lib/Rest/VstResponse.cpp

```diff
@@ -74,7 +73,7 @@ void VstResponse::addPayload(VPackSlice const& slice,
       VPackBuffer<uint8_t> tmpBuffer;
       tmpBuffer.reserve(slice.byteSize());  // reserve space already
       VPackBuilder builder(tmpBuffer, options);
-      VelocyPackHelper::sanitizeNonClientTypes(slice, VPackSlice::noneSlice(),
+       VelocyPackHelper::sanitizeNonClientTypes(slice, VPackSlice::noneSlice(),
```

Comment:
unrelated whitespace change, which we can revert
```suggestion
      VelocyPackHelper::sanitizeNonClientTypes(slice, VPackSlice::noneSlice(),
```

Reply:
Added.